app/models/concerns/team_rules.rb

```diff
@@ -21,6 +21,7 @@ def has_less_than_x_words(pm, value, _rule_id)
       return false unless pm.media&.type == 'Claim'
       smooch_message = get_smooch_message(pm)
       return false if smooch_message['text'].blank?
+      #TODO: Consider using Bot::Alegre.number_of_words
       smooch_message['text'].to_s.gsub(Bot::Smooch::MESSAGE_BOUNDARY, '').split(/\s+/).select{ |w| (w =~ /^[0-9]+$/).nil? }.size <= value.to_i
```

Comment:
I'm not familiar with this code and unsure whether we're better leaving it as is (i.e., white space separated tokens) or using Bot::Alegre.number_of_words. Any preference @caiosba ? If you're not sure, I'll leave as is and file a ticket in the backlog for further investigation in the future.

Reply:
Good catch. I think we can leave that code as is. It gets all messages from a conversation and ignores menu options first. But it used in rules which is not actively used (if used at all).